toplevel/coqtop.ml

```diff
@@ -121,6 +120,14 @@ let print_memory_stat () =
 
 let _ = at_exit print_memory_stat
 
+(******************************************************************************)
+(* Deprecated                                                                 *)
+(******************************************************************************)
+let _remove_top_ml () = Mltop.remove ()
```

Comment:
explain please

Reply:
This is in fact leftover code and removed in the next PR, sorry for the noise, will remove.

Reply:
You know, this refactoring wasn't easy XD I lost the count on how many times I reordered the code.